package.json

```diff
@@ -3,20 +3,22 @@
   "description": "A pty for xterm.js and Emscripten",
   "version": "0.10.2",
   "main": "index.js",
+  "types": "out/index.d.ts",
```

Comment:
This fixes the type definitions. Not relevant to the PR, but I noticed there was an issue open for this